tst/Devices/UltrasportBike/ultrasportbiketestdata.h

```diff
@@ -1,12 +1,12 @@
 ﻿#pragma once
 
-#include "Devices/bluetoothdevicetestdata.h"
+#include "Devices/Bike/biketestdata.h"
 #include "ultrasportbike.h"
 
-class UltrasportBikeTestData : public BluetoothDeviceTestData {
+class UltrasportBikeTestData : public BikeTestData {
 
 public:
-    UltrasportBikeTestData() : BluetoothDeviceTestData("Ultrasport Bike") {
+    UltrasportBikeTestData() : BikeTestData("Ultrasport Bike") {
```

Comment:
@cagnulein In this PR that I have abandoned for later significant rework, I made each test data class inherit from BikeTestData, EllipticalTestData or whichever in order to use inheritance.  With your approval, I would like to make this change as a separate PR.